dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.3445.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib llvm
+
+DESCRIPTION="LLVM-based OpenCL compiler targetting Intel Gen graphics hardware"
+HOMEPAGE="https://github.com/intel/intel-graphics-compiler"
+SRC_URI="https://github.com/intel/${PN}/archive/igc-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+COMMON="sys-devel/llvm:=[${MULTILIB_USEDEP}]
+	dev-libs/opencl-clang:=[${MULTILIB_USEDEP}]"
```

Comment:
On a note related to the above, this should exclude opencl-clang slots higher than LLVM_MAX_SLOT here.

Reply:
I don't think there' a way to restrict slot, other than explicitly listing them, so I got a bit creative with the version. It works as expected; changing LLVM_MAX_SLOT to 8 pulls in opencl-clang:8 and llvm:8.